### ISSUE TITLE
The details of the code

### ISSUE THREAD

**Issue by MaT-xuy** [roymiles/vkd#5]
Hello, your thoughts are very original giving me a lot of inspiration and ideas for deeper exploration, I would like to ask if the code can be run directly on the dataset CIFAR as well as the convenience of providing the details of the data in Figure 3?

**Comment by roymiles** [comment]
Hi, thank you for your comments! the T-SNE plot was created using [this](https://github.com/CannyLab/tsne-cuda).
We used a small CNN trained on MNIST and saved all the features just before the final linear layer.

```
tsne = TSNE(n_iter=1000, verbose=1, num_neighbors=64)
tsne_results = tsne.fit_transform(z.reshape(60000,-1))
```

The TSNE plot will look very similar if the features (in this case z) undergo any orthogonal transformation, but any arbitrary linear transformation will distort the plot a lot. Some of our [previous work](https://arxiv.org/pdf/2303.11098) looked into more details about a linear projector specifically and how it evolves during the distillation process.

The code in this repository does not directly support CIFAR training, so I would recommend using the [RepDistiller](https://github.com/HobbitLong/RepDistiller) repository. The CIFAR benchmark is quite saturated, so we used a few small tricks to get the best results, such as replacing the MSE loss function with a LogSum (see [here](https://github.com/roymiles/Simple-Recipe-Distillation/blob/31e8477cfdd6bb6ec7cdc332915fd4496b6de7b6/imagenet/torchdistill/losses/single.py#L159)).
